spec/praxis/endpoint_definition_spec.rb

```diff
@@ -21,7 +21,7 @@
     its([:description]) { should eq(endpoint_definition.description) }
     its([:media_type]) { should eq(endpoint_definition.media_type.describe(true)) }
 
-    its([:actions]) { should have(2).items }
+    its([:actions]) { should have(4).items }  # Two real actions, and twp post version of the GET
```

Comment:
Typo in comment, should be "Two real actions, and two post version of the GET"

Reply:
👍